python/ray/_private/usage/usage_lib.py

```diff
@@ -647,10 +647,10 @@ def get_instance_type(node_config):
         with open(cluster_config_file_path) as f:
             config = yaml.safe_load(f)
```

Comment:
@jjyao similar to Ricky's question above, I guess I should call `rewrite_deprecated_workers_fields` here as well? Let me know if you have any recommendations.  Happy to be more conservative to avoid breaking telemetry, but many users will continue to use both `max_workers` in their YAML files while some will migrate to `max_worker_nodes`, so let me know the best way to handle that

Reply:
@architkulkarni sorry for the late review. Yes, what you do here will break the telemetry and the server will reject the message since `min/max_worker_nodes` is a unknown field. We can rename the internal fields but for `UsageStatsToReport`, let's keep the old name for compatibility.